src/core/VarReplay.jl

```diff
@@ -346,6 +356,10 @@ function getranges(vi::VarInfo, spl::Sampler)
     end
 end
 
+function getranges(vi::VarInfo, s::Selector)
```

Comment:
Call this function in the `getranges` with `::Sampler` as a second argument to avoid duplication.

Reply:
Fixed by #726.